### ISSUE TITLE
Page modification request: http

### ISSUE THREAD

**Issue by fazlearefin** [tldr-pages/tldr#13487]
### Command description

Examples date back as long as 8 years. Examples should focus more on modern day use case like website and API debugging. 

### Platform

Common

**Comment by jxu** [comment]
Is this an alias for httpie?

**Comment by fazlearefin** [comment]
> Is this an alias for httpie?

Yes. This is the doc: https://httpie.io/docs/cli/usage

**Comment by acuteenvy** [comment]
No, it is not an alias. The `httpie` command is [the plugin manager](https://httpie.io/docs/cli/plugin-manager). Its page also needs an update, because it currently contains usage examples of the `http` command.

**Comment by fazlearefin** [comment]
Thanks @acuteenvy for pointing this out 

`http` and `httpie` are two separate command, and tldr page for `httpie` is wrong!

**Comment by fazlearefin** [comment]
Raised 2 separate PRs: #13507 and #13508
